atomicapp/constants.py

```diff
@@ -82,3 +82,10 @@
 
 # If running in an openshift POD via `oc new-app`, the ca file is here
 OPENSHIFT_POD_CA_FILE = "/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+
+# Index
+INDEX_IMAGE = "projectatomic/nulecule-library"  # temp until we have an official image on projectatomic docker hub
+INDEX_LOCATION = ".atomicapp/index.yaml"
+INDEX_NAME = "index.yaml"
+INDEX_DEFAULT_IMAGE_LOCATION = "localhost"
+INDEX_GEN_DEFAULT_OUTPUT_LOC = "./index.yaml"
```

Comment:
many of these variables essentially contain the same information. should we somehow have INDEX_NAME incorporated in the other ones that currently have `index.yaml` hardcoded?